.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2.1
+
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:2.6.5-buster-node-browsers
```

Comment:
Does this one line take care of all browser testing needs (i.e., running selenium_chrome_headless in a feature spec)?

Also, [this](https://circleci.com/docs/2.0/browser-testing/#using-screenshots-and-artifacts) seems like a great selling point

Reply:
yes. circle does us a kindness and has images that have browsers pre installed. 

`store_artifacts` is pretty rad. right now it's sending coverage up there, you should be able to see it on the job screen.

Reply:
That's really nice. Wrangling that in Travis is a pain.

Notice btw our suite is no longer 7 seconds, now a full minute plus just because of the feature specs. Still not too shabby though.

`store_artifacts` is interesting